sdk/communication/AzureCommunicationChat/AzureCommunicationChat.xcodeproj/xcshareddata/xcschemes/AzureCommunicationChat.xcscheme

```diff
@@ -45,7 +45,7 @@
          </EnvironmentVariable>
          <EnvironmentVariable
             key = "SDK_REPO_ROOT"
-            value = "/Users/travisprescott/repos/azure-sdk-for-ios"
+            value = "/Users/ningxinlan/Desktop/Dev/azure-sdk-for-ios"
```

Comment:
We probably want to blank this out to keep people's names out of this

Reply:
agree. will fix it.